test/functional/Java8andUp/src/org/openj9/test/utf8ClassPkgName/LoadClassWithUTF8PkgNameTest.java

```diff
@@ -0,0 +1,146 @@
+ * avoid any issue with platform dependencies.
+ * 
+ */
+@Test(groups = { "level.sanity" })
+public class LoadClassWithUTF8PkgNameTest {
+	
+	private static byte[] classBytes;
+	final static int classVersionJava8 = 52;
+	final static String loadedPackagePath = "org/openj9/resources/utf8pkgname/loaded_Utf8Class_\u6b63\u5728\u52a0\u8f7d\u7c7b\u5305\u540d\u957f\u5ea6\u6570\u68c0\u67e5/";
+	final static String loadedClassName = "Utf8ClassPackageNameTest";
+	final static String loadedClassFileWithPackagePath = loadedPackagePath + loadedClassName + ".class";
+	final static String loadedClassNameWithDotPath = loadedPackagePath.replace('/', '.') + loadedClassName;
+	final static String jarFileNameWithPath = System.getProperty("user.dir") + "/" + "Utf8PkgName.jar";
+	
+	public static byte[] generateClassBytes(String loadedPackagePath, String loadedClassName, int classVersion) throws Exception {
+
+		ClassWriter cw = new ClassWriter(0);
+		FieldVisitor fv;
+		MethodVisitor mv;
+		AnnotationVisitor av0;
+
+		cw.visit(classVersion, ACC_PUBLIC + ACC_SUPER, loadedPackagePath + loadedClassName, null, "java/lang/Object", null);
+		cw.visitSource(loadedClassName + ".java", null);
+
+		{
+			mv = cw.visitMethod(ACC_PUBLIC, "<init>", "()V", null, null);
+			mv.visitCode();
+			Label l0 = new Label();
+			mv.visitLabel(l0);
+			mv.visitLineNumber(29, l0);
+			mv.visitVarInsn(ALOAD, 0);
+			mv.visitMethodInsn(INVOKESPECIAL, "java/lang/Object", "<init>", "()V");
+			mv.visitInsn(RETURN);
+			Label l1 = new Label();
+			mv.visitLabel(l1);
+			mv.visitLocalVariable("this", "L" + loadedPackagePath + loadedClassName + ";", null, l0, l1, 0);
+			mv.visitMaxs(1, 1);
+			mv.visitEnd();
+		}
+		cw.visitEnd();
+
+		return cw.toByteArray();
+	}
+	
+	@BeforeTest
+	public void beforeTest() throws Exception {
+		String ver = System.getProperty("java.version");
+		boolean isJava8ClassFile = ((ver.length()>=3) && ver.substring(0, 2).equalsIgnoreCase("1.8")) ? true : false;
+		
+		/* The generated class file is only used by the java version >= Java 8 */
+		int classVersion = classVersionJava8;
+		if (!isJava8ClassFile) {
+			/* use the first located number in the version string assuming
+			 * the major version number is 9, 10, 11, 12, 13, 14 ... 
+			 * Note: internal build might come with '-' in the version string.
+			 * e.g. 14-internal
+			 */
+			String splitPattern = (ver.indexOf('.') >= 0) ? Pattern.quote(".") : 
+									((ver.indexOf('-') >=0) ? Pattern.quote("-"): null);
+			String verDigits = (null != splitPattern) ? ver.split(splitPattern)[0] : ver;
+			classVersion = (Integer.parseInt(verDigits) - 8) + classVersionJava8;
+		}
+		classBytes = generateClassBytes(loadedPackagePath, loadedClassName, classVersion);
+		
+		/* prepare the MANIFEST data for the specified jar file */
+		String jarFileVersion = "1.0.0";
+		String jarFileAuthor = "IBM OpenJ9";
+		Manifest manifestData = new Manifest();
```

Comment:
There is a JarOutputStream constructor that doesn't take a manifest. Can it be used instead to simplify the test?

Reply:
Removed as suggested above.